packages/docs/src/locales/en/en.json

```diff
@@ -2930,31 +2930,31 @@
     "description": "Vuestic UI components support keyboard interactions for all mouse-based actions and utilize HTML5 semantic elements where applicable.",
     "keyboardInteractions": {
       "title": "Keyboard accessible",
-      "description": "All the interactive elements of your page e.g. text boxes, buttons etc can be accessible by keyboard. That means you should be able to bring focus on them by using Tab and Shift+Tab keys."
+      "description": "All the interactive elements on your page (e.g. text boxes, buttons, etc.) can be accessed with the keyboard. That means you should be able to focus them by using Tab and Shift+Tab keys."
     }
   },
   "configurationGuide": {
     "title": "Configuration",
-    "description": "We have developed a convenient mechanism that allows you to deeply customize your icons, components, and colors.",
-    "readMore": "Read more",
+    "description": "Vuestic UI comes with a convenient mechanism that allows for a deep icons, components, and colors customization.",
```

Comment:
Could use if you like:
"Vuestic UI comes with a convenient mechanism that allows deep customization of icons, components, and colors."

Reply:
Just my writing style. Both are gramatically and correct :+1: